docs/databases/hadoop/how-to-install-and-set-up-hadoop-cluster/index.md

```diff
@@ -228,7 +232,7 @@ Edit `yarn-site.xml`:
 
     <property>
             <name>yarn.resourcemanager.hostname</name>
-            <value>node-master</value>
+            <value>203.0.113.0</value>
```

Comment:
since we're using a private IP with the hostname, we need to use the public IP in order for the yarn site to resolve